ccmlib/cluster.py

```diff
@@ -8,13 +8,15 @@
 from collections import OrderedDict, defaultdict
 from concurrent.futures import ThreadPoolExecutor
 
-import yaml
+from ruamel.yaml import YAML
```

Comment:
IIUC, a typical use case of preserving the comment in YAML file is when care about comment of an existing yaml file, but why would we care about the comment? do we read read the edited yaml file, or do we plan to use some script to extract the comments out of the edited yaml file and process them?

Reply:
i see. you are using ccm as a testbed for this feature we would be interested in, before replacing `yaml` with `ruamel.yam` in scylla-machine-image. but i don't understand how the test drive in this project helps to build the confidence of using it in another. is scylla-ccm sharing the same pipeline with scylla-machine-image?

i don't feel strong either way, as long as functionality wise, `ruamel.yam` is a drop-in replacement of `yaml`.

Reply:
it is a drop in replacement, 
and the main reason for this one, is to align all tests infrastructure to be able to return scylla.yaml as is, as much as possible.

it's doesn't have too much value on dtest or other users of ccm, since we don't even save those fails as artifacts (even that we probably should)